index.html

```diff
@@ -1768,42 +1698,34 @@ <h3 id="applying">
           structurally valid.
         </p>
         <p>
-          The <dfn>steps for processing the <code>lang</code> member</dfn> is
-          given by the following algorithm. The algorithm takes a
-          <var>manifest</var> as an argument. This algorithm returns a string
-          or <code>undefined</code>.
+          The <dfn>steps for post-processing the <code>lang</code> member</dfn>
+          is given by the following algorithm. The algorithm takes a
+          <a>WebAppManifest</a> <var>manifest</var> as an argument. This
+          algorithm returns a <code>DOMString?</code>.
         </p>
         <ol>
-          <li>Let <var>value</var> be the result of calling the
-          <a>[[\GetOwnProperty]]</a> internal method of <var>manifest</var>
-          with argument "<code>lang</code>".
+          <li>Let <var>V</var> be <var>manifest.lang</var>.
           </li>
-          <li>If <a>Type</a>(<var>value</var>) is not "string":
+          <li>If <a>Type</a>(<var>V</var>) is <code>String</code>:
```

Comment:
how come you changed these? I can live with them, but single letter attributes are not my favorite.

Reply:
I like them shorter, makes it easier for me to read, plus aligned with WebIDL spec

Reply:
> plus aligned with WebIDL spec

That's something I'm hoping to fix. It's a dreadful practice.

Reply:
I agree. It gets really confusing because you always need to go back to look up what something was. Also, the upper-case thing makes it look like a constant.

Reply:
OK guys I will change it :-) "value" it is